ReactiveXComponent/Connection/IXCSubscriber.cs

```diff
@@ -7,7 +7,7 @@ namespace ReactiveXComponent.Connection
     public interface IXCSubscriber : IDisposable
     {
         void Subscribe(string stateMachine, Action<MessageEventArgs> stateMachineListener);
-        void Unsubscribe(string stateMachine, Action<MessageEventArgs> stateMachineListener);
```

Comment:
If your interface offers the possibility to subscribe an Action, then it should offer a method to properly unsubscribe **that** particular Action.
You need to keep the _Unsubscribe(string stateMachine, Action<MessageEventArgs> stateMachineListener)_ methods and properly implement the unsubscription.